### PR TITLE
Fix errors in README.md

### PR DESCRIPTION
**Pull request by Rauf00**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
The README.md contains some grammatical errors and can be made more concise.

#### Before/After Screenshots/Screen Record

- Before:
- After:

#### Fixes the following issue(s)

- Fixes #

#### Relies on the following changes

- 

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [ ] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mhmdanas** [comment]
Thanks for the PR, but it seems like a lot of the changes aren't "error" fixes, but just subjective rewordings? Also, do please fill out the PR template.

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Whether you have ideas, translations, design changes, code cleaning, or even maj
 </a>
 
 ## Donate
-If you like NewPipe, you're welcome to send a donation. We prefer Liberapay, as it is both open-source and non-profit. For further info on donating to NewPipe, please visit our [website](https://newpipe.net/donate).
+If you like NewPipe, you're welcome to donate. We prefer Liberapay, as it is both open-source and non-profit. For further info on donating to NewPipe, please visit our [website](https://newpipe.net/donate).
```
Comment (mhmdanas) [inline README.md:118]: I can't quite put my finger on why, but I prefer the original wording.

**Review by mhmdanas — CHANGES_REQUESTED** [review]

**Comment by litetex** [comment]
Please fill out the template accordingly and reopen the PR when done so.

**Comment by opusforlife2** [comment]
Not only are some of the suggestions grammatically incorrect, some of them are removing important information. So I'm disinclined to accept this PR as it is.

You can come discuss with us in Newpipe's IRC about what changes you would like to see. It's pretty difficult to debate grammar through PR reviews.

**Pull request closed without merging**